### DISCUSSION TITLE
Better Server Storage View

### DISCUSSION THREAD

**Discussion by shahiljain** [immich-app/immich#14093] [Feature Request]
The server storage section currently looks like this,
![Screenshot_20241112-110148.png](https://github.com/user-attachments/assets/c6c0ff36-fdb9-4cce-ba9e-bb521ce7a8a6)

However it could give a bit more insight about how the storage is being used by sperating it into the following categories:
- My Photos
- Other Users
- Others (Non Immich)

A crude rendition of what it could look like:
![Picsart_24-11-12_11-15-43-324.jpg](https://github.com/user-attachments/assets/5b24a38a-648b-4e56-907a-52e3e42652c0)

This could be useful to see on a quick glance how much storage you and your users are using.

**Comment by shahiljain** [comment]
Also I know the numbers don't add up and the category ratios are nonsense ... I used a mobile photo editor to quickly come up with this.

**Comment by nckrwlmn** [comment]
Also allow us to set a custom full storage size. Or turn off the storage size altogether. So it would just show GB used. If the filesystem is being used for other things, the used percentage doesnt reflect what is going on.

**Comment by guyfischman** [comment]
I would also like to see broken down the usage for transcoded videos/photos, database, and immich software, so that I can get a total for immich within my server (this could replace the current 'STORAGE SPACE' in Server Stats which presently only reflects non-transcoded user usage).

**Comment by bo0tzz** [comment]
#3186
